src/sasjs-help/index.js

```diff
@@ -137,6 +137,21 @@ export async function printHelpText() {
           command example: sasjs context list --target targetName
           command example: sasjs context list -t targetName
           NOTE: Providing target name (--target targetName or -t targetName) is optional. Default target name will be used if target name was omitted.
+        * ${chalk.cyanBright(
+          'export'
+        )} - exports context to contextName.json in the current folder.
+          command example: sasjs context export contextName --target targetName
+          command example: sasjs context export contextName -t targetName
+          NOTE: Providing target name (--target targetName or -t targetName) is optional. Default target name will be used if target name was omitted.
+
+          exported context example:
+            {
+              "createdBy": "admin",
+              "links": [...],
```

Comment:
This isn't so helpful - we need to export a context that has the same JSON format as we use to create and modify contexts.  The idea is this:

```
sasjs context export contextName
## change ONLY name attribute inside the exported context
sasjs context create --source=contextName.json
## change ONE attribute
sasjs context edit contextName --source=contextName
```
This will provide the most intuitive and user-friendly experience.  To clarify:

* `sasjs context export contextName`
* `sasjs context create`
* `sasjs context edit contextName` 

Should all use the *same* JSON format, and that format would ideally be listed as a type in the Adapter TypeScript